class/defaults.yml

```diff
@@ -1,3 +1,11 @@
 parameters:
   metallb:
-    namespace: syn-metallb
+    namespace: metallb-system
```

Comment:
Why? Unless there are technical reasons, we  only use namespaces starting with `syn-`.